results/feeds.py

```diff
@@ -19,24 +24,31 @@ class BasicResultEventsFeed(Feed):
     description = _("A basic feed of election results")
 
     def items(self):
-        return ResultEvent.objects.filter(created__gte="2017-05-04")
+        date_limit = datetime.now() - timedelta(days=14)
+        return ResultEvent.objects.filter(created__gte=date_limit) \
```

Comment:
In theory `current` should come from Every Election, and by default that marks them as not current 20 days after the election. Of course, the importer isn't running at the moment, so the elections will never be marked as not current. 

I still think that it's better to have a singe source of 'currentness' rather than having different bits of the site have different ideas. If someone marked an election as not current before the 14 days, I expect there would be a good reason for it.In theory `current` should come from Every Election, and by default that marks them as not current 20 days after the election. Of course, the importer isn't running at the moment, so the elections will never be marked as not current. 

I still think that it's better to have a singe source of 'currentness' rather than having different bits of the site have different ideas. If someone marked an election as not current before the 14 days, I expect there would be a good reason for it.